### ISSUE TITLE
Keyboard focus isn't in the main text field when app is launched

### ISSUE THREAD

**Issue by MarcoZehe** [notepad-plus-plus-mac/notepad-plus-plus-macos#34]
When launching NotePad++ 1.0.5, e. g. with CMD+O in Finder, keyboard focus is not set on the actual text field. I am a VoiceOver user, and it is reported that focus is on one of the horizontal or vertical splitters. I have to navigate to the scroll area and interact with it to get to the FileZilla control. The control itself works fine once focus is in there, but initially, keyboard focus is somewhere else. The result is that after launching the app, I cannot start typing immediately.

**Comment by aletik** [comment]
The fix is based into version 1.0.6, coming in a week.
